src/contexts/UserContext.js

```diff
@@ -8,7 +8,7 @@ export default UserContext;
 
 export const UserContextProvider = ({ children }) => {
   const [isUserAuthenticated, setIsUserAuthenticated] = useState(
-    getAccessToken() !== null && getRefreshToken() !== null && getUserId() !== null
+    (getAccessToken() !== null || getRefreshToken() !== null) && getUserId() !== null
```

Comment:
I think both access and refresh token should be not null though. What was the rationale behind changing it to `||`